seleniumDescriptors.js

```diff
@@ -0,0 +1,177 @@
+var format = require('util').format;
```

Comment:
Ah. The Intern projects tend to lean more towards larger modules; code isn't generally split into separate modules unless it's shared, there is a significant readability advantage, or testability is improved. In this case it seems kind of odd to split out the config files since they're not being individually tested and aren't relevant to (or used by) anything but the Selenium tunnel.